stylesheets/styles.css

```diff
@@ -17,6 +17,13 @@ body, html {
     background-color: beige;
 }
    
+.CatchTheSunLogo {
+    display: flex;
+    height: 210%;
+    width: 860px;
+    padding-top: 3px;
+}
```

Comment:
Thoughts: 

- Could we possibly change the class name to "logo" instead of "CatchTheSunLogo" to make it short and simple?
-  In trying to keep the stylesheet uniform, could we change the use of 'px' to 'em' or 'rem'?